ts/input/tex/ParseOptions.ts

```diff
@@ -173,6 +174,14 @@ export default class ParseOptions {
       list = this.nodeLists[property] = [];
     }
     list.push(node);
+    if (node.kind !== property) {
+      //
+      // If the list is not just for its kind, record that it is in this list
+      //   so that if it is copied, the copy can also be added to the list.
+      //
+      let lists = (NodeUtil.getProperty(node, 'in-lists') as string || '').split(',').concat(property).join(',');
+      NodeUtil.setProperty(node, 'in-lists', lists);
```

Comment:
This attribute makes it to the output. If it should, could it get a data-* prefix?

Reply:
I'm not able to reproduce that.  Can you give an example that does it for you?

Reply:
`\begin{equation} F(z,N) := e^{f(z,N)} := e^{z/N}N^{l-1}(1-e^{z/N})^{l-1}\prod _{j=1}^N \frac{1}{1-e^{zj/N}}, \tag{4}\end{equation}` gets me `in-lists=",fixStretchy"` on the `g` around the `:=`

Reply:
Just the := is enough, actually.

Reply:
OK, thanks.  It turns out that this is a very special case, where two `mo` elements are combined into one.  The code that does this uses a function to copy all the internal properties from one of them to the combined `mo`, but unfortunately that function sets the attributes rather than the internal properties.  I will make a PR to fix it.  I've check the other uses of the function, and this appears to be the only place that occurs.